README.md

```diff
@@ -5,23 +5,63 @@
 
 ### Requirements
 
-    - xargo or cargo xbuild
-    - xtensa-esp32-elf toolchain must be in your `$PATH`
-    - esptool, if targeting the esp32 or esp8266
+#### llvm-xtensa
+Please refer to the [llvm-xtensa](https://github.com/espressif/llvm-xtensa) project for authoratative instructions.
```

Comment:
That project is archived, https://github.com/espressif/llvm-project is the new one

Reply:
Thanks!